changelogs/unreleased/gh-7904-export-box_schema_version-to-public-api-and-lua.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/box
```

Comment:
The documentation request holds several requirements:

> 0 is returned before server configuration (i.e., box.cfg) or during
recovery and the old value of the schema version is returned in
transactions (i.e., DDL operations in transactions do not change the schema
version). Rolled back DDL operations also do not change the schema version.

As I see, neither of them are tested.

Reply:
Discussed with @Totktonada, @alyapunov, @unera: the this patch does not intend to change the behaviour of schema version number, so it remains unspecified with its monotonic growth being the only requirement held.